fuzzers/binary_only/qemu_cmin/src/fuzzer.rs

```diff
@@ -137,17 +133,38 @@ pub fn fuzz() -> Result<(), Error> {
         ))
     };
 
+    let stdout_callback = |buf: &[u8]| {
+        if let Ok(s) = from_utf8(buf) {
```

Comment:
you could write it raw (`io::stdout().lock();`) instead of utf8-ing back and forth (would be better for perf)

Other option would be to print the buf hexlified if it doesn't decode (`"{buf:X?}"`)

But up to you

Reply:
I might just leave as is for now if that's ok? We can revisit later if it becomes a problem though. Just want to get the PR over the line so I can see if there's anything that can be done about the CI issues.